### ISSUE TITLE
Pre-train VGG on different application

### ISSUE THREAD

**Issue by 32nguyen** [zsdonghao/SRGAN#37]
Hi zsdonghao,
Thanks for sharing your work with tensorflow. On my application, my inputs do not have 3 channels like RGB images (let says they have 5 channels). I believe that I can not use the pre-trained VGG because my inputs have different standard.

So, my concern is that if I want to apply pre-trained VGG, I have to pre-train it first with my inputs (5 channels) until certain time, then use those weights as a pre-trained VGG like in the paper. I hope this is a correct way.

Bests,

**Comment by zsdonghao** [comment]
Hi @32nguyen , in you case, the simplest way is to remove VGG loss, but in my experience, it may produce checkerboard artifact.

If you want to use perceptual loss, you can do what you say above: ``pre-train a CNN first with your inputs (5 channels) until certain time, then use this CNN as a pre-trained model like in the paper.``

**Comment by 32nguyen** [comment]
Great! Thank you.
